doozerlib/cli/__main__.py

```diff
@@ -1126,6 +1125,15 @@ def images_mirror_streams(runtime, streams, dry_run):
             brew_image = config.image
             brew_pullspec = runtime.resolve_brew_image_url(brew_image)
             cmd = f'oc image mirror {brew_pullspec} {upstream_dest}'
+            registry_config_file = ''
+            if runtime.registry_config is not None:
+                for f in os.listdir(runtime.registry_config):
+                    if f.endswith(".json"):
+                        registry_config_file = os.path.join(runtime.registry_config, f)
+                        break
+                if registry_config_file == '':
+                    raise FileNotFoundError("Can not find the registry config file in {}".format(runtime.registry_config))
+                cmd += " --registry-config={}".format(runtime.registry_config)
```

Comment:
you wanted the file here, not the directory.
```suggestion
                cmd += f" --registry-config={registry_config_file}"
```

otherwise you get:
```
$ DOCKER_CONFIG=/workspaces/doozer doozer -g openshift-3.11 -i configmap-reload images:push --to-defaults\
...
distgit.py:606:rc, out, err = exectools.cmd_gather(mirror_cmd) : $5: ['oc', 'image', 'mirror', '--insecure=true', '--filename=/workspaces/doozer-working-dir/push/configmap-reload', '--registry-config=/workspaces/doozer'] - [cwd=/workspaces/doozer-working-dir/distgits/containers/configmap-reload]: Executed:cmd_gather
2020-08-10 22:54:17,880 INFO [containers/configmap-reload] Error mirroring image -- retrying in 60 seconds.
error: unable to load --registry-config: read /workspaces/doozer: is a directory
```